_posts/2021-05-28-status-update.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: "2021-05-28: IAC & DevX Team Status Update"
+author: carabasdaniel
+categories:
+  - team
+  - status
+  - developer
+tags:
+---
+
+We had a bit of a slow week today. We have a few company-wide mental health holidays and one was this week. It's really good to disconnect from it all sometimes. 
+
+## Community Contributions
+
+We'd like to thank the following people in the Puppet Community for their contributions over this past week:
+
+- [`puppetlabs-mysql#1387`][puppetlabs-mysql-pr-1387]: "Update xtrabackup package name for Ubuntu 20.04", thanks to [rsynnest][rsynnest]
+- [`iac#252`][iac-pr-252]: "Bump nokogiri from 1.11.0 to 1.11.5", thanks to [dependabot][dependabot]
+
+Check [all the ways to reach us]({% link _posts/2021-01-20-reaching-out.md %}) if you want to directly contact us about anything module related.
+
+## New Module / Gem Releases
+
+The following modules were released this week:
+
+- [`puppetlabs-docker`][puppetlabs-docker] (`4.0.1`)
+- [`puppetlabs-apache`][puppetlabs-apache] (`6.2.0`)
+- [`puppetlabs-postgresql`][puppetlabs-postgresql] (`7.2.0`)
+- [`puppetlabs-haproxy`][puppetlabs-haproxy] (`6.0.1`)
+- [`puppetlabs-tomcat`][puppetlabs-tomcat] (`5.1.0`)
+- [`puppetlabs-vsphere`][puppetlabs-vsphere] (`2.0.0`)
+
+  [puppetlabs-docker]: https://github.com/puppetlabs/puppetlabs-docker
+  [puppetlabs-apache]: https://github.com/puppetlabs/puppetlabs-apache
+  [puppetlabs-postgresql]: https://github.com/puppetlabs/puppetlabs-postgresql
+  [puppetlabs-haproxy]: https://github.com/puppetlabs/puppetlabs-haproxy
+  [puppetlabs-tomcat]: https://github.com/puppetlabs/puppetlabs-tomcat
+  [puppetlabs-vsphere]: https://github.com/puppetlabs/puppetlabs-vsphere
+  [puppetlabs-mysql-pr-1387]: https://github.com/puppetlabs/puppetlabs-mysql/pull/1387
+  [rsynnest]: https://github.com/rsynnest
+  [iac-pr-252]: https://github.com/puppetlabs/iac/pull/252
+  [dependabot]: https://github.com/apps/dependabot
+
+### DSC And the PDK Go Prototype
+
+With the DevX team [hard at work on a new approach to PDK templating][pdkgo-templating] (and the release of their work imminent), this felt like a good week to see what it will be like to use the new templating for a real project.
+We put up a [prototype PR][puppet-dsc-pdkgo-template] for moving from switching to the new templating (if available) in our [`Puppet.Dsc`][puppet-dsc-intro-blog] module.
```

Comment:
```suggestion
The DevX team is [hard at work on a new approach to PDK templating][pdkgo-templating] and their LA release is imminent), so this week felt like a good time to see what it will be like to use the new templating in a real project.

We put up a [prototype PR][puppet-dsc-pdkgo-template] for switching to the new templating (when it becomes available) in our [`Puppet.Dsc`][puppet-dsc-intro-blog] module.
```